aws/resource_aws_dax_cluster_test.go

```diff
@@ -239,9 +275,9 @@ EOF
 func testAccAWSDAXClusterConfig(rString string) string {
 	return fmt.Sprintf(`%s
 		resource "aws_dax_cluster" "test" {
- 		  cluster_name       = "tf-%s"
+		  cluster_name       = "tf-%s"
 		  iam_role_arn       = "${aws_iam_role.test.arn}"
-		  node_type          = "dax.r3.large"
+		  node_type          = "dax.t2.small"
```

Comment:
Node type changed as encryption at rest is not supported for dax.r3.* node types:
* https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DAXEncryptionAtRest.html

Reply:
👍